src/main/java/HW13/Company.java

```diff
@@ -0,0 +1,18 @@
+package HW13;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
```

Comment:
Do you need setters? You have annotation @acessors. The same about all the rest entities.